client/index.html

```diff
@@ -83,185 +84,186 @@ <h1 class="title">The Lounge is loading…</h1>
 										<input type="checkbox" name="tls" {{#if defaults.tls}}checked{{/if}} {{#if lockNetwork}}disabled{{/if}}>
-										Enable TLS/SSL
+										<span data-translate="true">index.connect.enable_tls_ssl</span>
 									</label>
 								</div>
 								<div class="clearfix"></div>
 							</div>
 							{{/if}}
 							<div class="col-sm-12">
-								<h2>User preferences</h2>
+								<h2 data-translate="true">index.connect.user_preferences</h2>
 							</div>
 							<div class="col-sm-3">
-								<label for="connect:nick">Nick</label>
+								<label data-translate="true" for="connect:nick">index.connect.nick</label>
 							</div>
 							<div class="col-sm-9">
 								<input class="input nick" id="connect:nick" name="nick" value="{{defaults.nick}}">
 							</div>
 							{{#unless useHexIp}}
 							<div class="col-sm-3">
-								<label for="connect:username">Username</label>
+								<label data-translate="true" for="connect:username">index.connect.username</label>
 							</div>
 							<div class="col-sm-9">
 								<input class="input username" id="connect:username" name="username" value="{{defaults.username}}">
 							</div>
 							{{/unless}}
 							<div class="col-sm-3">
-								<label for="connect:realname">Real name</label>
+								<label data-translate="true" for="connect:realname">index.connect.real_name</label>
 							</div>
 							<div class="col-sm-9">
 								<input class="input" id="connect:realname" name="realname" value="{{defaults.realname}}">
 							</div>
 							<div class="col-sm-3">
-								<label for="connect:channels">Channels</label>
+								<label data-translate="true" for="connect:channels">index.connect.channels</label>
 							</div>
 							<div class="col-sm-9">
 								<input class="input" id="connect:channels" name="join" value="{{defaults.join}}">
 							</div>
 							<div class="col-sm-9 col-sm-offset-3">
-								<button type="submit" class="btn">Connect</button>
+								<button data-translate="true" type="submit" class="btn">index.connect.connect</button>
 							</div>
 						</div>
 					</form>
 				</div>
 				<div id="settings" class="window">
 					<div class="header">
-						<button class="lt" aria-label="Toggle channel list"></button>
+						<button class="lt" aria-label="index.toggle_chan_list"></button>
 					</div>
 					<div class="container">
 						<div class="row">
 							<div class="col-sm-12">
-								<h1 class="title">Settings</h1>
+								<h1 class="title" data-translate="true">index.settings.title</h1>
 							</div>
 							<div class="col-sm-12">
-								<h2>Messages</h2>
+								<h2 data-translate="true">index.settings.messages</h2>
 							</div>
 							<div class="col-sm-6">
 								<label class="opt">
 									<input type="checkbox" name="motd">
-									Show <abbr title="Message Of The Day">MOTD</abbr>
+									<span data-translate="true">index.settings.show_motd</span> <abbr title="Message Of The Day">MOTD</abbr>
```

Comment:
Currently `index.settings.show_motd` is just "Show", and then the `abbr` is put after that. This works fine with English, but it would be better if the `abbr` is inserted into the translation string, rather than tacked on at the end